datasets/alt/alt.py

```diff
@@ -0,0 +1,407 @@
+}
+
+_SPLIT = {
+    "train": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/URL-train.txt",
+    "dev": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/URL-dev.txt",
+    "test": "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/URL-test.txt",
+}
+
+_WIKI_URL = "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/ALT-Parallel-Corpus-20191206/URL.txt"
+
+
+class AltConfig(datasets.BuilderConfig):
+    """BuilderConfig for ALT."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for ALT.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(AltConfig, self).__init__(**kwargs)
+
+
+class Alt(datasets.GeneratorBasedBuilder):
+    """Asian Language Treebank (ALT) Project"""
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="alt", version=datasets.Version("1.0.0"), description="Asian Language Treebank Parallel Corpus"
+        ),
+        # datasets.BuilderConfig(
+        #     name="alt-en", version=datasets.Version("1.0.0"), description="English ALT"
+        # ),
+        # datasets.BuilderConfig(
+        #     name="alt-jp", version=datasets.Version("1.0.0"), description="Japanese ALT"
+        # ),
+        # datasets.BuilderConfig(
+        #     name="alt-my", version=datasets.Version("1.0.0"), description="Myanmar ALT"
+        # ),
+        # datasets.BuilderConfig(
+        #     name="alt-km", version=datasets.Version("1.0.0"), description="Khmer ALT"
+        # ),
+        # datasets.BuilderConfig(
+        #     name="alt-my-transliteration", version=datasets.Version("1.0.0"), description="Myanmar-English Transliteration Data Set"
+        # ),
+        # datasets.BuilderConfig(
+        #     name="alt-my-west-transliteration", version=datasets.Version("1.0.0"), description="Latin-Myanmar Transliteration Dataset"
+        # )
+    ]
+
+    DEFAULT_CONFIG_NAME = "alt"
+
+    def _info(self):
+        if self.config.name == "alt":
+            features = datasets.Features(
+                {
+                    "SNT.URLID": datasets.Value("string"),
+                    "SNT.URLID.SNTID": datasets.Value("string"),
+                    "url": datasets.Value("string"),
+                    "bg": datasets.Value("string"),
+                    "en": datasets.Value("string"),
+                    "en_tok": datasets.Value("string"),
+                    "fil": datasets.Value("string"),
+                    "hi": datasets.Value("string"),
+                    "id": datasets.Value("string"),
+                    "ja": datasets.Value("string"),
+                    "khm": datasets.Value("string"),
+                    "lo": datasets.Value("string"),
+                    "ms": datasets.Value("string"),
+                    "my": datasets.Value("string"),
+                    "th": datasets.Value("string"),
+                    "vi": datasets.Value("string"),
+                    "zh": datasets.Value("string"),
```

Comment:
Could you use the `Translation` feature type here ?

You can look at [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py) for an example of usage.

You'll need to regenerate the dataset_infos.json after this change

Reply:
Because it consists of 13 languages which are translated from English wiki.
Should I list language pairs only from English to XX or all possible combinations? The latter seems to be non-practical as the authors plan to add more data so the language pairs will increase N*(N-1).